src/libcmd/command.cc

```diff
@@ -211,60 +207,58 @@ Strings editorFor(const Path & file, uint32_t line)
 {
     auto editor = getEnv("EDITOR").value_or("cat");
     auto args = tokenizeString<Strings>(editor);
-    if (line > 0 && (
-        editor.find("emacs") != std::string::npos ||
-        editor.find("nano") != std::string::npos ||
-        editor.find("vim") != std::string::npos ||
-        editor.find("kak") != std::string::npos))
+    if (line > 0
+        && (editor.find("emacs") != std::string::npos || editor.find("nano") != std::string::npos
+            || editor.find("vim") != std::string::npos || editor.find("kak") != std::string::npos))
         args.push_back(fmt("+%d", line));
     args.push_back(file);
     return args;
 }
 
 MixProfile::MixProfile()
 {
-    addFlag({
-        .longName = "profile",
-        .description = "The profile to update.",
-        .labels = {"path"},
-        .handler = {&profile},
-        .completer = completePath
-    });
+    addFlag(
+        {.longName = "profile",
```

Comment:
The change to the formatting of designated initializers causes a huge diff, but I don't see a way to keep the old style...

Reply:
We could use [`.git-blame-ignore-revs`](https://github.com/community/community/discussions/5033#discussioncomment-2318478) as [we do in `nixpkgs`](https://github.com/NixOS/nixpkgs/blob/master/.git-blame-ignore-revs).